core/eolearn/core/core_tasks.py

```diff
@@ -555,6 +557,32 @@ def map_method(self, feature):
         return feature[..., self.bands]
 
 
+class ExplodeBandsTask(EOTask):
+    """Explode a subset of bands from one feature to multiple new features."""
+
+    def __init__(
+        self,
+        input_feature: Tuple[FeatureType, str],
+        output_features: Iterable[Tuple[FeatureType, str]],
+        bands: Iterable[List[int]],
```

Comment:
this way of specifying input is kind of problematic. What if the two iterables don't match? What if the user forgets an element (and the rest are off by one) etc.

I think it would make more sense to have a mapping (via dict or similar) of band indices to output features.

Reply:
you can switch to using tuples for indices

Reply:
but having it the other way around is also completely fine (names to indices)

Reply:
We'll have it names to indices.